### DISCUSSION TITLE
What are the specific formatting settings on the special tags you're supporting

### DISCUSSION THREAD

**Discussion by Gewerd-Strauss** [chrisgrieser/shimmering-focus#249] [Q&A]
Hello,

I have been working on adapting my own set of colorful tags, which are mostly just modifications to those you supply so far. I usually use methods either in singular, or nest it and append the specific method I require, e.g. method for generalised info, methods/LC-MSMS for the specific method of `Liquid Chromatography-Tandem Mass-Spectrometry`.

Issue is, I'd like to match the normal tag formatting as closely as possible, but I cannot find info on what curvature you're applying.

This is the current state of the art on my side, and I cannot ignore the few pixels around the corners that I cannot seem to get right. 
![grafik](https://user-images.githubusercontent.com/80548781/219847071-81497ca7-9510-436f-8a2b-53285b222be7.png)

I am assuming that the underlying setup is compatible with `Colorful Tags`, both because you are refering to it with regards to making your own tags, and because the result aside from the border-form is actually spot-on. 

---

Thank you.  
Sincerely,  
~Gw

**Comment by chrisgrieser** [comment]
you can take a look how the annotation tags are implemented here in my theme: https://github.com/chrisgrieser/shimmering-focus/blob/main/source.css#L815-L991

Tt looks like a lot, but that's cause it's multiple tags. One tag alone should be ~10 lines of code. 

Btw, you should  definitelycheck out the Obsidian discord, if you aren't there already. especially in the `#appearance` channel you'll find various other people for asking/discussing CSS stuff like this
